modules.d/95rootfs-block/mount-root.sh

```diff
@@ -30,21 +30,12 @@ mount_root() {
         fsck_ask_err
     done
 
-    READONLY=
     fsckoptions=
     if [ -f "$NEWROOT"/etc/sysconfig/readonly-root ]; then
```

Comment:
Old RH pre-systemd stuff: https://access.redhat.com/solutions/2196881
I'd remove the `/etc/sysconfig/readonly-root` part as well.